sql-ballerina/tests/complex-query-test.bal

```diff
@@ -273,6 +275,105 @@ function testDateTime() {
     test:assertEquals(value, expected, "Expected record did not match.");
 }
 
+type ResultDates2 record {
+    time:Date DATE_TYPE;
+    time:TimeOfDay TIME_TYPE;
+    time:Civil TIMESTAMP_TYPE;
+    time:Civil DATETIME_TYPE;
+    time:TimeOfDay TIME_TZ_TYPE;
+    time:Civil TIMESTAMP_TZ_TYPE;
```

Comment:
Shall we change field names to lowercase?

Reply:
I will update